src/DomCrawler/Crawler.php

```diff
@@ -173,7 +173,7 @@ public function attr($attribute): ?string
             return $this->text();
         }
 
-        return $element->getAttribute($attribute);
+        return $element->getAttribute($attribute) ?? '';
```

Comment:
Why not this?
```suggestion
        return $element->getAttribute($attribute) ?: '';
```

Reply:
Because a ternary is less explicit? Another solution is just to cast `return (string) $element->getAttribute($attribute));` It's maybe more elegant.

Reply:
Totally agree (even though it's merged :p )